src/Microsoft.ML.Data/Scorers/SchemaBindablePredictorWrapper.cs

```diff
@@ -426,9 +426,15 @@ private protected override bool SaveAsOnnxCore(OnnxContext ctx, RoleMappedSchema
 
             var mapper = ValueMapper as ISingleCanSaveOnnx;
             Contracts.CheckValue(mapper, nameof(mapper));
-            Contracts.Assert(schema.Feature.HasValue);
             Contracts.Assert(Utils.Size(outputNames) == 3); // Predicted Label, Score and Probablity.
 
+            // Prior doesn't have a feature column and uses the training label column to determine predicted labels
+            if (mapper.ToString().EndsWith("PriorModelParameters")) {
+                var labelColumnName = schema.Schema[0].Name;
+                return mapper.SaveAsOnnx(ctx, outputNames, ctx.GetVariableName(labelColumnName));
```

Comment:
Can you use the same pattern as schema.Feature.HasValue? (schema.Label.HasValue)?

Reply:
I can check whether the name is null, but the schema itself doesn't have the label name initialized. BinaryPredictionTransformer only populates the schema's feature column.